src/core/meshcop/meshcop_tlvs.cpp

```diff
@@ -0,0 +1,72 @@
+/*
```

Comment:
We have `meshcop/meshcop_tlvs.cpp` and `meshcop/tlvs.hpp`.  Prefer to have the filenames match between the `.cpp` and `.hpp` files.